NBA/getGames.py

```diff
@@ -0,0 +1,56 @@
+from nba_api.stats.static import teams,players
+from nba_api.stats.endpoints import leaguegamefinder
+import pandas as pd
+
+team_columns = ['TEAM_ID', 'GAME_ID',
+       'GAME_DATE', 'MATCHUP', 'WL']
+
```

Comment:
This file does everything and it should be broken up.

You should have one file called `nba_api_helper.py` which contains a class `NBA_ApiHelper` which has functions like `get_games` and gets the raw list of games form NBA API based on query parameters. 

Then you should have a file that has functions to clean and reformat the data in preparation for computing elo. This file needs tests. 

Lastly, you can have a separate file with functions to write your resultant data to the database when you update the elos. This takes in the raw data and returns the data in a friendly format for computing ELOs.

This allows you to test the code that does the data clean without being dependent on APIs.